internal/docs/Operator-CRD.md

```diff
@@ -2,19 +2,23 @@
 
 ### Use Prometheus Pod and Service Monitor Custom Resources
 The Azure Monitor metrics add-on supports scraping Prometheus metrics using Prometheus - Pod Monitors and Service Monitors, similar to the OSS Prometheus operator. Enabling the add-on will deploy the Pod and Service Monitor custom resource definitions to allow you to create your own custom resources.
-Creating these custom resources allows for easy configuration of scrape jobs in any namespace, especially useful in the multi tenancy scenario, where partners don’t have access to the kube-system namespace and one error prone scrape job might affect the other partners scrape jobs. This is also true without the multitenancy model. Currently, when there is an erroneous scrape job in the custom config map, all the jobs in custom config map are ignored and the addon uses just the default scrape jobs. 
```

Comment:
@sunasing - Why are these changes in a separate repo? Lets plan to keep all the docs we are sharing with customers at the same place. Could you please add/edit the doc in our repo? https://github.com/Azure/prometheus-collector/blob/main/internal/docs/Operator-CRD.md